content/blog/vs-firefox-relay.md

```diff
@@ -111,4 +129,24 @@ Using email aliases isn't mainstream (yet) and has probably the same popularity
 
 Having reputed companies investing in the email aliases technology like Apple and Firefox also validates the need for email aliases.
 
+{{< rawhtml >}}
+<br>
+  <div style="padding: 35px; background-color: #f8f9fa; border-radius: 15px;">
+    <img src="/logo.svg" style="height:35px;"/>
+    <br><br><br>
+    <p class="mb-2 h5"><strong>The best email aliasing service</strong></p>
+    <p class="mb-2">
+        Thanks to SimpleLogin <strong>browser extensions and mobile apps</strong> (iOS & Android), create email aliases from all of your devices. You can even use SimepleLogin's Firefox add-on.
```

Comment:
"You can even use SimepleLogin's Firefox add-on" -> "SimpleLogin is also available as a `<a href="https://addons.mozilla.org/firefox/addon/simplelogin"/>Firefox add-on</a>`